ui/src/tasks/actions/index.ts

```diff
@@ -3,9 +3,9 @@ import {push, goBack} from 'react-router-redux'
 import _ from 'lodash'
 
 // APIs
-import {LogEvent, ITask as Task} from '@influxdata/influx'
 import {client} from 'src/utils/api'
 import {notify} from 'src/shared/actions/notifications'
+import * as api from 'src/client'
```

Comment:
I think it's a really bad idea for us to use two different clients to interact with fetching/writing the same resource(tasks). If we are switching to `src/client` for tasks let's make sure each function is swapped with one that is functionally equivalent.

Reply:
I hear ya. My reasoning behind not refactoring the entire `client` library was because I didn't want this to get too out of scope (which is ironic in a way since it did). I can go ahead and finish refactoring this file if you think that'd be prudent

Reply:
I'm sorry this got out of hand! I think we should go the other direction and add defaults to check and notification rule Labels instead of removing them

Reply:
No worries at all. @desa fully warned me, I just underestimated it. Would you mind elaborating a bit on what you mean by going in the other direction?